### ISSUE TITLE
driver not found

### ISSUE THREAD

**Issue by jasonspick** [bencarter78/odbc#4]
I am getting a driver not found error. I've done some research and from what I can tell, php pdo for odbc comes pre configured. So why am I getting this error??

**Comment by reno1979** [comment]
In Laravel 5.3 I encountered problems (Driver not Found) with this plugin. 
This plugin extends the available database drivers and should therefor be called right after the line

Illuminate\Database\DatabaseServiceProvider::class,

in config/app.php

**Comment by rafwell** [comment]
have extension=php_pdo_odbc.dll in your php.ini?

**Comment by surgiie** [comment]
im have the same problem :/

**Comment by jacksonrnunes** [comment]
Im have the same problem in laravel 5.2.45...
Solutions?

**Comment by rafwell** [comment]
make sure you have the php-odbc extension installed.
on php_info you need see PDO Drivers => mysql, odbc, sqlite .... (odbc must be here)

**Comment by jacksonrnunes** [comment]
Im fix the problem. It was not PDO. I did not update on the composer.
Im newbie to laravel and composer, and did not know I needed to do update after installing a new package.
I could put this in the documentation for the newbies ones myself. =)

**Comment by rafwell** [comment]
You need change composer.json to add a new dependencie, in this case, bencarter78/odbc. After this, you need run composer update to recheck the dependencies and download the new package.

**Comment by jacksonrnunes** [comment]
Yes! I discovered this as looking at installing other packages ...
Thank´s!
